lib/std/sysrand.nim

```diff
@@ -0,0 +1,304 @@
+#
+#
+#            Nim's Runtime Library
+#        (c) Copyright 2021 Nim contributors
+#
+#    See the file "copying.txt", included in this
+#    distribution, for details about the copyright.
+#
+
+## `std/sysrand` generates random numbers from a secure source provided by the operating system.
```

Comment:
the acronym CSPRNG should also be mentioned somewhere in this file for searchability